Terraform/terraform-deployment.yml

```diff
@@ -0,0 +1,46 @@
+name: "Terraform Deploy"
+on:
+  push:
+    branches:
+      - master
```

Comment:
Can you please include comments to list the secret configuration to be done by the user to use this template? I understand that you have called this out in the readme, but for better usability of the template, it is ideal that you include this detail within the template. Thanks!

Reply:
Ideally the readme should suffice but I understand your point that since these are starter workflows, the users can be fairly new to github.